examples/tutorials/self-paced-training/part-3_security_and_privacy/chapter-6_Security_in_federated_compute_system/06.3_site_security_privacy_policy/site_policy.ipynb

```diff
@@ -0,0 +1,475 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ef224801-e14b-4b8e-92bb-1643364fbef9",
+   "metadata": {},
+   "source": [
+    "# Site-specific security and privacy policies\n",
+    "\n",
+    "NVIDIA FLARE offers a set of security mechansim to control the user access for different opertions based on their roles. These control not limited to centralized server control, but also at each site. Such site-specific level of security is done via security policies which we call federated polices. Probably should called site-specific security & privacy polices.\n",
```

Comment:
suggest:
"NVIDIA FLARE offers a set of security mechanisms to control user access for different operations based on their roles. These controls are not limited to centralized server control but also apply at each site. This site-specific level of security is managed through security policies, which we refer to as federated policies."

If we really want, we need to rename all federated policies -> federated security and privacy policies throughout our documentation